compose.yml

```diff
@@ -0,0 +1,19 @@
+# This file is used to run a local instance of
```

Comment:
heh - i did not know they changed this filename convention. TIL

Reply:
I just recently saw that too. Now that I'm looking at the [docs](https://docs.docker.com/compose/compose-application-model/#the-compose-file), it says `compose.yaml` is preferred but either should work 🤔